src/compliance/dto/trusted-claim-issuer.dto.ts

```diff
@@ -10,10 +10,11 @@ export class TrustedClaimIssuerDto {
       'List of Claim types for which an Identity is trusted for verifying. Defaults to all types',
     enum: ClaimType,
     isArray: true,
+    nullable: true,
   })
   @IsOptional()
   @IsEnum(ClaimType, { each: true })
-  readonly trustedFor?: ClaimType[];
+  readonly trustedFor?: ClaimType[] | null;
```

Comment:
```suggestion
  readonly trustedFor: ClaimType[] | null = null;
```
If this gets defaulted to null, then we don't have to deal with the undefined case, which means a type assertion isn't needed in the service.